ch_pipeline/hfb/containers.py

```diff
@@ -10,12 +10,40 @@
 
 from ch_util import andata
 
-from ..core.containers import RawContainer, FreqContainer
+from ..core.containers import ContainerBase, RawContainer, FreqContainer
 
 from draco.core.containers import TODContainer
 
 
-class HFBData(RawContainer, FreqContainer):
+class HFBContainer(ContainerBase):
+    """A base class for all HFB containers.
+
+    Like :class:`ContainerBase`, but with some properties specific to HFB data.
+    """
+
+    @property
+    def hfb(self) -> memh5.MemDataset:
+        """The main hfb dataset."""
+        return self.datasets["hfb"]
```

Comment:
Given that you're centralising the logic here, you might want to actually improve on it and offer a more useful exception message if this lookup fails.

Reply:
Done.